cli/src/main/java/io/kestra/cli/commands/migrations/TenantMigrationService.java

```diff
@@ -0,0 +1,21 @@
+package io.kestra.cli.commands.migrations;
+
+import com.github.javaparser.utils.Log;
+import io.kestra.core.repositories.TenantMigrationInterface;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import lombok.extern.slf4j.Slf4j;
+
+@Singleton
+@Slf4j
+public class TenantMigrationService {
+
+    @Inject
+    private TenantMigrationInterface tenantMigrationInterface;
+
+    public void migrateTenant(String tenantId, String tenantName, boolean dryRun) {
+        Log.info("🔁 Starting tenant migration...");
+        tenantMigrationInterface.migrateTenant("main", dryRun);
```

Comment:
You have a constant for that I think in the TenantService?

The best would be also to check the if `tenantId` is not null and != main you throw with a message only main tenant is supported in OSS.
Same if tenantName is not null you throw